WordPress/Classes/ViewRelated/NUX/SigninWPComViewController.swift

```diff
@@ -0,0 +1,279 @@
+import UIKit
+import WordPressComAnalytics
+import WordPressShared
+
+/// Provides a form and functionality for signing a user in to WordPress.com
+///
+@objc class SigninWPComViewController : NUXAbstractViewController, SigninWPComSyncHandler, SigninKeyboardResponder
+{
+
+    @IBOutlet weak var usernameField: WPWalkthroughTextField!
+    @IBOutlet weak var passwordField: WPWalkthroughTextField!
+    @IBOutlet weak var submitButton: NUXSubmitButton!
+    @IBOutlet weak var statusLabel: UILabel!
+    @IBOutlet var bottomContentConstraint: NSLayoutConstraint!
+    @IBOutlet var verticalCenterConstraint: NSLayoutConstraint!
+    var onePasswordButton: UIButton!
+
+    var immediateSignin = false;
+
+    lazy var loginFacade: LoginFacade = {
+        let facade = LoginFacade()
+        facade.delegate = self
+        return facade
```

Comment:
I don't think you're setting the delegate to `nil` on `deinit` so this happens:

1. Launch the app and enter your username
2. Enter whatever you want in the password field
3. Tap the Sign In button and as soon as possible tap Back
4. You get back to the Email screen and get an error. When you dismiss the error the screen is white.

![screen shot 2016-04-12 at 21 47 37](https://cloud.githubusercontent.com/assets/8739/14473484/a290fbb8-00f8-11e6-93c8-27d2cf38d492.png)